GiniVisionExample/GiniVisionExample.xcodeproj/project.pbxproj

```diff
@@ -236,6 +269,8 @@
 			knownRegions = (
 				en,
 				Base,
+				de,
+				es,
```

Comment:
We don't want Spain :*

Reply:
I'm using only it for testing on my device, but It would not be an issue having an additional language. But yes, I will remove it because It could lead to issues with someone developing in this project.